go.mod

```diff
@@ -3,16 +3,14 @@ module go.temporal.io/api
 go 1.16
 
 require (
-	github.com/gogo/googleapis v1.4.1
-	github.com/gogo/protobuf v1.3.2
-	github.com/gogo/status v1.1.1
 	github.com/golang/mock v1.6.0
-	github.com/golang/protobuf v1.5.3
-	github.com/grpc-ecosystem/grpc-gateway v1.16.0
+	github.com/google/go-cmp v0.5.9
+	github.com/grpc-ecosystem/grpc-gateway/v2 v2.18.0
+	github.com/iancoleman/strcase v0.3.0
```

Comment:
Can you inline (poor-man's vendor) this dependency? I would usually never request such a thing, but how this library is used in so many projects coupled with the very limited need for our use is a bit too left-pad-ish.

Reply:
I can if we really want to. I'll add it to my list

Reply:
FWIW, if we're talking about the risk of the GitHub repo disappearing some day (like left-pad) did; I was told the other day in our Slack that all Go dependencies are cached on Google's proxy and disappearing modules shouldn't be an issue 🤷

Reply:
I did not make the request due to "disappearing modules", I made it to reduce the dependencies on our downstream users. `go.temporal.io/api` is a library is used by hundreds/thousands of users, so there needs to be a good reason to increase its transitive dependency surface. This isn't one IMO.

The main issue with increasing these dependencies is not size or anything, it's versioning. We are now forcing everyone to have v0.3.0 of this library. v0.2.0 and v0.4.0 are not compatible with this version (see https://go.dev/ref/mod#versions) so we have now put unnecessary rules on our users which is bad.

Reply:
Once I have the server's unit tests passing (I'm almost there) I'll just write the two functions (ToSnake, ToCamel) we need myself.